### ISSUE TITLE
v1.2.10 WebGL Build not Connecting

### ISSUE THREAD

**Issue by newbiedev1** [MoralisWeb3/unity-web3-game-kit#128]
I created new empty project and installed 1.2.10 game kit. Wallet connection is working perfectly on Unity Editor with metamask scanning.
Unity version: 2021.3.9f1 apple silicon
Game kit version: 1.2.10
After that, I switched target to WebGL. I changed to moraliswebgl template on Player settings.
So i build webgl and open with chrome. No errors i have. But when i press connect button, connected my metamask wallet. But after connect wallet, there is only black screen. Is this normal?

https://user-images.githubusercontent.com/112906910/189605056-e5fcdfa6-0d8f-4cb5-ae64-88c8701ff91e.mp4

**Comment by newbiedev1** [comment]
My device: Macbook Air M2 Monterey 12.6
Unity Version: 2021.3.9f1 SILICON LTS
Game kit version: 1.2.10
I created a new empty project and i changed to WebGL. After that, i installed moralisweb3sdk_v1_2_10.unitypackage file to my new empty project.
![image|690x395](upload://1h9u3lXwXHwWqmj7QiJ6eYqH8Kk.jpeg) 
After that:

https://user-images.githubusercontent.com/112906910/190520576-b59cc2c3-bf4f-45f3-bb16-e07c91bbb73c.mp4

https://user-images.githubusercontent.com/112906910/190520834-305cbbc5-e80b-457c-9726-869b98b7de02.mp4

**Comment by KoenRijpstra** [comment]
Thanks for the detail information. Could you see if there are any errors in Chrome developer console? https://developer.chrome.com/docs/devtools/open/

**Comment by newbiedev1** [comment]
> https://developer.chrome.com/docs/devtools/open/

<img width="1459" alt="image" src="https://user-images.githubusercontent.com/112906910/190977565-c2a62022-b5e5-4422-bb5a-ff4f13308f18.png">

**Comment by newbiedev1** [comment]
Black screen problem is still same with v1.2.11. Problem not solved on WebGL Build.

**Comment by KoenRijpstra** [comment]
Could you check if the `x-parse-application-id` header is getting send? Looks like the same problem as #125

**Comment by newbiedev1** [comment]
> Could you check if the `x-parse-application-id` header is getting send? Looks like the same problem as #125

I found the problem about black screen
Black screen issue persists on Apple Silicon Builds. Wallet link not working. But macbooks with Intel processors do not have such a problem. I tried your build on my macbook air m2 (apple silicon) and it has black screen problem issue. But i tried it with macbook pro that has intel processor, it is working perfectly.
